src/app.js

```diff
@@ -1,21 +1,34 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const http = require('http');
+const fs = require('fs');
+const url = require('url');
 
-module.exports = sum;
+const PORT = 5700;
+
+const server = http.createServer((req, res) => {
+  const base = req.url.slice(1).split('/');
+  const checkedNameFile = base.includes('file')
+    && req.url.slice(1).replace(/file/, 'public');
+  const asd = checkedNameFile || req.url.slice(1);
+
+  const normalizedUrl = new url.URL(asd, `http://${req.headers.host}`);
+  const renameFile = normalizedUrl.pathname.slice(1)
+    .replace(/public/, '').split('/')
+    .filter(i => i !== '').join('/');
```

Comment:
If you want to filter only truthy values you can use the next approach:
```
.filter(Boolean)
``` 
It's shorter and clearer.